app/views/users/_profile.html.erb

```diff
@@ -1,8 +1,10 @@
 <%= javascript_packs_with_chunks_tag "colorPreview", "validateFileInputs", defer: true %>
 
-<div class="crayons-card mb-6 p-5">
-  <%= render "users/additional_authentication" %>
-</div>
+<% if @user.identities.size < 2 %>
```

Comment:
didn't know we added this too!

can you use `if user.identities.count < SiteConfig.authentication_providers.count` ? I shall add a custom method soon

Reply:
Ohh yeah that's great 👍

Reply:
I'm going to stick with `@user.identities.size` or `.count` unless you think otherwise?

Reply:
no that's fine!

Reply:
BTW in `/account` there's the same problem so it's probably easier to add it to the partial itself